### ISSUE TITLE
har2locust returns error

### ISSUE THREAD

**Issue by stlxiii** [locustio/locust#2364]
### Describe the bug
This is not for this repo exactly, but I have found no other avenue for support with Locust or har2locust. Please if there is a support contact or a mailing list let me know :)
Problem: 
Most of the .har files I attempt to import fail with: error: cannot format -: 'utf-8' codec can't decode byte [...]
I have attached a specific example I'm trying to use.
[test.zip](https://github.com/locustio/locust/files/11790569/test.zip)

### Expected behavior
No decoding error.

### Actual behavior
cannot format -: 'utf-8' codec can't decode byte [...]

### Steps to reproduce
extract the file I have attached and:
har2locust test.har

### Environment

- OS: Windows 10
- Python version: 3.10

**Comment by cyberw** [comment]
Perhaps your har file is not UTF-8 encoded, but ISO-8859-1? Did you maybe open your file in an editor and save it again?

I guess we should make make har2locust try different encodings (or at very least give a more descriptive error message)
https://stackoverflow.com/questions/19699367/for-line-in-results-in-unicodedecodeerror-utf-8-codec-cant-decode-byte

**Comment by stlxiii** [comment]
As far as I know the file is UTF-8. It was generated by Edge. I've imported it successfully but I had to remove from it all occurrences of the character "é" and another character which I forget what it was (some quotation mark).

**Comment by cyberw** [comment]
I’ll try to have a look this weekend.

**Comment by stlxiii** [comment]
The other character that was causing problems is: ’
Ex:
> Prenez note **qu’une** signature est requise

**Comment by Lewiscowles1986** [comment]
The file is utf-8 according to Visual Studio Code; I Also used that encoding filter to change to with and without BOM UTF8 and got the following `sha256sum` outputs:

- 004e07dfc07cfea736aa9d8063be9f570c65bbfa740f468e8a136b5a3e4fee62 (original file after unzip)
- d6add88a9f24582395935ebc3b9013ef582ca0037de64d69d00e6edff564b9b2 (with utf-8 including BOM)
- 004e07dfc07cfea736aa9d8063be9f570c65bbfa740f468e8a136b5a3e4fee62 (post save again as utf-8 without BOM)

In the past I've had similar issues with CSV files opening under python where they were not malformed at all; and I just switched to using io to load the file, and specifying the encoding. Can't remember how long ago this was though, all I remember was needing to `import io` and use with `io.open` and that it overcame a utf-8 problem. (sorry for lack of details there)

**Comment by cyberw** [comment]
I can convert this file fine on both macos and windows. Maybe the problem was related to WSL?

And you are right Lewis, this should be filed in har2locust not in locust.

**Comment by stlxiii** [comment]
I logged this here because

1. It's in the locust documentation to use har2locust
2. There is no way to create an issue in the har2locust repo.

I don't understand why this is closed, is there a resolution?

**Comment by cyberw** [comment]
Its ok for it to be here, but the code is in a separate repo so ideally it should be there. And your report is very welcome. Unfortunately I cannot reproduce it. Were you using WSL?

**Comment by stlxiii** [comment]
I'm just using a pretty standard setup. Windows 10 in English (although the website I'm testing is in French, hence the accented characters).  Python 3.10.8.  I'm using a virtual environment for the packages of each project so it's as clean an environment as possible.

When you import the example I uploaded, it works without error?

**Comment by cyberw** [comment]
Yup. The files work just fine on my Win11 machine (swedish, python 3.11) as well as on my mac. Perhaps you can look in to fixing it yourself?

**Comment by Lewiscowles1986** [comment]
@stlxiii if you were to install a hex editor, such as HxD; I am totally interested, even if I have not been able to reproduce.

**Comment by cyberw** [comment]
Perhaps the zip compress/decompress fixes it somehow? Are you running with the original file or the packed and then unpacked file?

**Comment by stlxiii** [comment]
I tested this morning with the zip attached here since I no longer have the original file.
